### ISSUE TITLE
Bug| x3d | Head Without Batch Normalization 

### ISSUE THREAD

**Issue by MoH-assan** [facebookresearch/pytorchvideo#254]
Hi 
The line below should be update to something like this, right? 

`pre_norm_module = None if norm is None else norm(num_features=dim_inner, eps=norm_eps, momentum=norm_momentum)`
https://github.com/facebookresearch/pytorchvideo/blob/eb04d1b21e08cfd0713164c0907aeb4c98fd83af/pytorchvideo/models/x3d.py#L482
